djangocms_bootstrap4/contrib/bootstrap4_link/constants.py

```diff
@@ -15,3 +15,9 @@
     ('', _('Medium')),
     ('btn-lg', _('Large')),
 )
+
+LINK_ICONS = getattr(
```

Comment:
Maybe a better name for this var? At first sight I thought it was a list, not a boolean.

Reply:
@vxsx suggestion sounds good (`USE_LINK_ICONS`)